test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -161,6 +163,16 @@ class CheckAllAnswersAndSubmitControllerSpec
       .expects(cgtReference, chargeReference, amount, returnUrl, backUrl, *, *)
       .returning(EitherT.fromEither[Future](response))
 
+  def mockCgtRegistrationService(representeeCgtReference: RepresenteeCgtReference) =
+    (mockSubscriptionService
+      .registerWithoutIdAndSubscribe(
+        _: CompleteRepresenteeAnswers
+      )(
+        _: HeaderCarrier
+      ))
+      .expects(*, *)
```

Comment:
you could still pass in `CompleteRepresenteeAnswers` to assert that the correct thing is being passed into the service and do something like `.expects(expectedAnswers, *)`. 

Also, you could pass in the response of the service like `Either[Error,RepresenteeCgtReference]` so you can do something like `.returning(EitherT.fromEither(response))` to allow you to test when things go wrong.